src/pages/index.tsx

```diff
@@ -33,7 +33,7 @@ const Home: FunctionComponent = () => (
     <HomepageHeader />
     <main className={styles.container}>
       <Screen maxWidth="wide">
-        <Grid gapVertical="small" paddingVertical="medium">
+        <Grid paddingTop="medium" paddingBottom="large" className="ams-grid_outer">
```

Comment:
I think the class `ams-grid_outer` can be removed, I don't see any class/styles related with this name when I inspect in devtools.